Sources/Rendering/Core/ImageCPRMapper/index.d.ts

```diff
@@ -157,6 +158,59 @@ export interface vtkImageCPRMapper extends vtkAbstractMapper3D {
 	 */
 	setDirectionMatrix(mat: mat3): boolean;
 
+	/**
+	 * Enable or disable Maximum Intensity Projection
+	 * Can be configured to use other modes of projection
+	 * @see getMipSlabThickness
+	 * @see getMipNumberOfSamples
+	 * @see getMipMode
+	 */
+	getUseMip(): boolean;
+
+	/**
+	 * @see getUseMip
+	 * @param useMip
+	 */
+	setUseMip(useMip: boolean): boolean;
```

Comment:
couldn't it be automatically assumed by the number of samples ? (if 1 it means it uses mip)